src/shader_recompiler/ir/ir_emitter.h

```diff
@@ -96,6 +104,33 @@ class IREmitter {
     void StoreBufferFormat(int num_dwords, const Value& handle, const Value& address,
                            const Value& data, BufferInstInfo info);
 
+    [[nodiscard]] Value BufferAtomicIAdd(const Value& handle, const Value& a, const Value& b,
+                                         BufferInstInfo info);
+    [[nodiscard]] Value BufferAtomicSMin(const Value& handle, const Value& a, const Value& b,
```

Comment:
If BufferAtomicIMin exists with is_signed argument, having separate BufferAtomicSMin/BufferAtomicUMin function is unnecessary, so you can remove them and select which opcode to emit directly from IMin